src/components/Navbar.elements.js

```diff
@@ -1,7 +1,9 @@
+import { DropdownButton } from 'react-bootstrap';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9172775)
```suggestion
import { DropdownButton } from "react-bootstrap";
```